frontend/src/components/atoms/FormField/Select.styled.ts

```diff
@@ -22,3 +22,16 @@ export const StyledLabel = styled.label`
   font-size: 20px;
   color: ${colors.background.darkGray};
 `;
+
+export const StyledSocialWrapper = styled.div`
+  display: inline-flex;
+  width: 100%;
+  align-items: center;
+`;
+
+export const StyledSocialIcon = styled.img`
+  width: 24px;
+  height: 24px;
+  margin-right: 10px;
+  cursor: pointer;
+`;
```

Comment:
Are you sure this styling is correctly located? Should be inside Social component?